doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1468,26 +1468,32 @@ Numeral notations
    parsing and printing functions, respectively.  The parsing function
    :n:`@ident__2` should have one of the following types:
 
-     * :n:`Decimal.int -> @ident__1`
-     * :n:`Decimal.int -> option @ident__1`
-     * :n:`Decimal.uint -> @ident__1`
-     * :n:`Decimal.uint -> option @ident__1`
+     * :n:`Numeral.int -> @ident__1`
+     * :n:`Numeral.int -> option @ident__1`
+     * :n:`Numeral.uint -> @ident__1`
+     * :n:`Numeral.uint -> option @ident__1`
      * :n:`Z -> @ident__1`
      * :n:`Z -> option @ident__1`
-     * :n:`Decimal.decimal -> @ident__1`
-     * :n:`Decimal.decimal -> option @ident__1`
+     * :n:`Numeral.numeral -> @ident__1`
+     * :n:`Numeral.numeral -> option @ident__1`
 
    And the printing function :n:`@ident__3` should have one of the
    following types:
 
-     * :n:`@ident__1 -> Decimal.int`
-     * :n:`@ident__1 -> option Decimal.int`
-     * :n:`@ident__1 -> Decimal.uint`
-     * :n:`@ident__1 -> option Decimal.uint`
+     * :n:`@ident__1 -> Numeral.int`
+     * :n:`@ident__1 -> option Numeral.int`
+     * :n:`@ident__1 -> Numeral.uint`
+     * :n:`@ident__1 -> option Numeral.uint`
      * :n:`@ident__1 -> Z`
      * :n:`@ident__1 -> option Z`
-     * :n:`@ident__1 -> Decimal.decimal`
-     * :n:`@ident__1 -> option Decimal.decimal`
+     * :n:`@ident__1 -> Numeral.numeral`
+     * :n:`@ident__1 -> option Numeral.numeral`
+
+     .. deprecated:: 8.12
+        Numeral notations on ``Decimal.uint``, ``Decimal.int`` and
```

Comment:
Here we say
```rst
``Decimal.uint``
```
whereas below we say
```rst
:g:`Decimal.uint`
```
Does it matter?  Should we standardize?  cc @coq/doc-maintainers

Reply:
Let's be uniform anyway.